.buildkite/browser-pipeline.yml

```diff
@@ -90,7 +90,7 @@ steps:
         concurrency: 5
         concurrency_group: "bitbar-web"
 
-      - label: ":edge: v106 Browser tests"
+      - label: ":edge: Browser tests"
```

Comment:
`:edge:` will be transformed into the Edge icon by Buildkite, so I don't know if `:edge: Latest Edge browser tests` would be clearly to read in the Buidlkite UI?  It's up to you, I would just make the wording consistent for each such job in the pipeline (i.e. Firefox/Chrome as well).